helm/ako/templates/configmap.yaml

```diff
@@ -26,6 +26,8 @@ data:
   {{ if .Values.AKOSettings.syncNamespace  }}
   syncNamespace: {{ .Values.AKOSettings.syncNamespace | quote }}
   {{ end }}
+  nsSyncLabelKey: {{ .Values.AKOSettings.nameSpaceSelector.labelKey | quote }}
+  nsSyncLabelValue: {{ .Values.AKOSettings.nameSpaceSelector.labelValue | quote }}
```

Comment:
May be I wasn't clear about the label configuration. But can we make it a bit structured here, like:
nsSyncLabel: |
   key:
   value: